src/language/extension-methods.md

```diff
@@ -220,6 +220,7 @@ extension NumberParsing on String {
 
 The members of an extension can be methods, getters, setters, or operators.
 Extensions can also have static fields and static helper methods.
+Make sure to use the Extension name for invoking them.
```

Comment:
Could you explain further what you are trying to suggest here? Are you suggesting using the Extension name explicitly to call the methods? Such as `NumberParsing('42').parseInt()`?

We do cover when you need to do in [Using extension methods](https://dart.dev/language/extension-methods#using-extension-methods) and its [API conflicts](https://dart.dev/language/extension-methods#api-conflicts) subcategory.